Content.Server/Fax/FaxSystem.cs

```diff
@@ -300,6 +301,11 @@ private void OnToggleInterface(EntityUid uid, FaxMachineComponent component, Aft
         UpdateUserInterface(uid, component);
     }
 
+    private void OnFileButtonPressed(EntityUid uid, FaxMachineComponent component, FaxFileMessage args)
+    {
+        PrintFile(uid, component, args);
+    }
```

Comment:
Server must validate all inputs from client, there's no validation going on so client could send war and peace to the server and it would print it.

Reply:
Sorry, I didn't get what you meant.

Reply:
Server should be truncating the input rather than relying on client to send it as they could send a million words.